lib/ProductOpener/Nutriscore.pm

```diff
@@ -707,6 +739,8 @@ sub compute_nutriscore_score_2023 ($nutriscore_data_ref) {
 				$nutriscore_data_ref->{$nutrient . "_points"}++;
 			}
 		}
+
+		$nutriscore_data_ref->{$nutrient . "_max"} = scalar @{$points_thresholds_2023{$nutrient_threshold_id}};
```

Comment:
`_max` is a bit misleading here, maybe `max_points` would be more easy to understand.

Reading this code I though you made a mistake by not taking the last value of the array.

At least we should put a comment:
```suggestion
		# max number of squares in the gauge we will display to user for this nutrient
		$nutriscore_data_ref->{$nutrient . "_max"} = scalar @{$points_thresholds_2023{$nutrient_threshold_id}};
```

Reply:
Good point, I renamed max to points_max